datacommons/examples/places_basic.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Basic demo for get_places_in
+
+"""
+
+import datacommons as dc
+
+def print_header(label):
+  print('\n' + '-'*80)
+  print(label)
+  print('-'*80 + '\n')
+
+def main():
```

Comment:
I feel we might want to have the basic and pandas as one example. Overall, we might not want to emphasis too much on the two options. We will mention the capability but not making it feel too overwhelming (too many options for user)

Reply:
Yeah I agree, I've merged the examples together into a single file.